cmd/vela-artifactory/main.go

```diff
@@ -40,148 +40,173 @@ func main() {
 	app.Flags = []cli.Flag{
 
 		&cli.StringFlag{
-			EnvVars: []string{"PARAMETER_LOG_LEVEL", "VELA_LOG_LEVEL", "ARTIFACTORY_LOG_LEVEL"},
-			Name:    "log.level",
-			Usage:   "set log level - options: (trace|debug|info|warn|error|fatal|panic)",
-			Value:   "info",
+			EnvVars:  []string{"PARAMETER_LOG_LEVEL", "VELA_LOG_LEVEL", "ARTIFACTORY_LOG_LEVEL"},
+			FilePath: string("/vela/secrets/parameter/log_level,/vela/secrets/artifactory/log_level"),
```

Comment:
Curious to know what we need the `/vela/secrets/parameter` path for?

I like the idea of `/vela/secrets/<plugin_name>` like you have for the second path 👍

Reply:
It was mostly added to keep the consistency with the `EnvVars` section. I was trying to model it off where we supported both the parameters injection and the secrets equivalently. Overall, the more I added it to plugins I hated it because of potential collisions that occur. The only flag across all the plugins I felt there was a potential use case was the log level. _Maybe_ someone wants to have verbose logs across all the plugins but it doesn't feel like a great use case. 

If you don't like it, I can remove it because I also don't like it. I'd rather start with just `/vela/secrets/<plugin_name>/` and build the other use cases we need.

Reply:
Thanks for explaining! I think that really helps me understand where you're coming from 😃 

Now I'm curious to know your thoughts on this one... and we can keep it as is because I agree that there is a consistency factor here.

But what if we changed the path just slightly? Keep in mind that I'm really throwing an idea out there and haven't thought about all the pros/cons and maybe you already thought of this and had a reason why it wouldn't work.

What I propose is we change the path to be `/vela/parameters/<plugin_name>/*`?

The reason why I'm proposing a slightly different variation is it could be a little confusing to have the `parameters` nested under the `secrets` path since they aren't really secrets.

I guess, I'm thinking that in a step you can have both `parameters` and `secrets` at the same indentation level. So maybe it makes sense for each of those keys to have their own path in the volume?

i.e.

```yaml
steps:
  - name: copy_artifacts
    image: target/vela-artifactory:v0.2.0
    pull: true
    secrets: [ artifactory_username, artifactory_password ]
    parameters:
      action: copy
      path: libs-snapshot-local/foo.txt
      target: libs-snapshot-local/bar.txt
      url: http://localhost:8081/artifactory
```

Reply:
I like that actually! The secrets plugin can read to any so they could still store generic values and read them in or it gives the ability to write from the pipeline to a location and have our plugins support them. I’m thinking that fit some of the customer who have been asking for a dynamic path in Artifactory plugin specifically.